iot-hub/Samples/device/DeviceReconnectionSample/ColorConsoleLogger.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Extensions.Logging;
+using System;
+using System.Globalization;
+using System.Linq;
+
+namespace Microsoft.Azure.Devices.Client.Samples
+{
+    public class ColorConsoleLogger : ILogger
```

Comment:
Most of these classes lack meaningful comments

Reply:
true, I'll add proper comments